litmus-portal/frontend/src/views/CreateWorkflow/ChooseWorkflow/index.tsx

```diff
@@ -184,13 +187,15 @@ const ChooseWorkflow: React.FC = () => {
       </div>
       <Unimodal isOpen={open} handleClose={() => setOpen(false)} hasCloseBtn>
         <Typography className={classes.modalHeading} display="inline">
-          Create your <strong>workflow name</strong>
+          <Trans i18nKey="createWorkflow.chooseWorkflow.modalHeading">
+            Create your <strong>workflow name</strong>
+          </Trans>
```

Comment:
`Trans` is only used for interpolation and in our use case, we are taking the language for a yam file with the `t` function. Is there any practical use-case for `Trans` here? If no, can we remove this please!

Reply:
I wasn't using the `Trans` component for interpolation purposes but rather to integrate React nodes in translated text which lets us translate the entire formatted text in one go as mentioned in the [docs](https://react.i18next.com/latest/trans-component). With `t`, we would have to maintain two separate entires in the yaml file, one for "Create your" and one for "workflow name" here. Should I still change all of them to `t`?

Reply:
But in the docs they mention that it does only interpolation and does not render [https://react.i18next.com/latest/trans-component](https://react.i18next.com/latest/trans-component)

Reply:
Ok then. I'll change them to `t`.